src/Field/styled.ts

```diff
@@ -0,0 +1,36 @@
+import { rgba } from 'polished';
+import styled from 'styled-components';
+import { get } from 'styled-system';
+import { StyledInput } from '../Input/styled';
+
+export const FieldLabel = styled('label')`
+  font-weight: ${p => p.theme.fontWeights.medium};
+  line-height: ${p => p.theme.lineHeights.body};
+`;
+
+export const FieldRequired = styled.span`
+  margin-left: ${p => p.theme.space.xs}px;
+  color: ${p => get(p.theme.colors, 'red.600')};
+`;
+
+export const FieldControl = styled('div')<{ isInvalid?: boolean }>`
+  ${StyledInput} {
+    ${p =>
+      p.isInvalid
+        ? `
+    box-shadow: ${rgba(get(p.theme.colors, 'red.600'), 0.83)} 0px 0px 0px 3px;
```

Comment:
I prefer using a lighter box-shadow color and a solid red border over here. Same for the focused inputs but with blue. This is just my take on the design though, so feel free to play around as you see fit.